README.md

```diff
@@ -655,6 +655,17 @@ const chart = new KeenDataviz({
   })
 ```
 
+### Chart render only when visible
+
+```javascript
+const chart = new KeenDataviz({
+    container: '#some_container', // required
+    chart: {
+      renderOnVisible: true
+    }
```

Comment:
there's no need to nest this property, since config is all about the `chart`, let's change to:
```javascript
const chart = new KeenDataviz({
    container: '#some_container', // required
    renderOnVisible: true
})
```